book.rb

```diff
@@ -0,0 +1,17 @@
+require './rental'
+
+# Book class
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] `rental` should be marked with `attr_accessor`, as we intend to both _access (get)_ as well as _modify (set)_ it. Kindly ensure to improve upon this aspect.